platform/pensando/dsc-drivers/src/drivers/linux/eth/ionic/ionic.h

```diff
@@ -22,10 +22,10 @@ struct ionic_lif;
 #define PCI_DEVICE_ID_PENSANDO_IONIC_ETH_VF	0x1003
 #define PCI_DEVICE_ID_PENSANDO_IONIC_ETH_MGMT	0x1004
 
-#define DEVCMD_TIMEOUT  5
+#define DEVCMD_TOUT_DEF 5
+#define DEVCMD_TIMEOUT  devcmd_timeout
```

Comment:
Is this kind of remapping really needed? Can the references of `DEVCMD_TIMEOUT` be replaced with the variable?

Reply:
All the code changes inside dsc-drivers are made by our drivers team and they keep updating it here in the open source repo
https://github.com/pensando/dsc-drivers/blob/master/drivers/